pkgs/applications/networking/irc/thelounge/default.nix

```
@@ -35,10 +35,12 @@ stdenv.mkDerivation (finalAttrs: {
 
   offlineCache = fetchYarnDeps {
     yarnLock = "${finalAttrs.src}/yarn.lock";
-    hash = "sha256-MM6SgVT7Pjdu96A4eWRucEzT7uNPxBqUDgHKl8mH2C0=";
+    hash = "sha256-csVrgsEy9HjSBXxtgNG0hcBrR9COlcadhMQrw6BWPc4=";
   };
 
-  nativeBuildInputs = [ nodejs yarn fixup-yarn-lock python3 npmHooks.npmInstallHook ] ++ lib.optional stdenv.hostPlatform.isDarwin cctools;
+  # Distutils was deprecated in 3.10, and removed in 3.12. This build needs it. An alternative could be adding
+  # setuptools, but testing with that and 3.12 still fails.
+  nativeBuildInputs = [ nodejs yarn fixup-yarn-lock python311 npmHooks.npmInstallHook ] ++ lib.optional stdenv.hostPlatform.isDarwin cctools;
```

Opinionated, but visually and semantically better:

```suggestion
  nativeBuildInputs = [ nodejs yarn fixup-yarn-lock python311 npmHooks.npmInstallHook ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ cctools ];
```